### DISCUSSION TITLE
[archived] "Custom Links" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12605] [General]
# en/latest/links/

https://argo-workflows.readthedocs.io/en/latest/links/

**Comment by apatel69** [comment]
Is it possible to access pod parameters for pod links? For example inputs provided to a specific pod step?

```
${step.inputs.param_name}
```

**Reply by Joibel** [reply]
Why not try it and report back?

**Reply by agilgur5** [reply]
Not exactly in the way you're trying. You have access to the `Workflow` [object](https://github.com/argoproj/argo-workflows/blob/6a39edf366319a40d37ccf406fe27dcee3d15705/ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx#L226), so you may need to parse through that to get the parameter of a specific step

**Comment by quom** [comment]
Is it possible to make these links open in a new tab? It's not possible to middle-click and it's infuriating to navigate away from the workflow page every time

**Reply by quom** [reply]
I discovered that if you press ctrl/meta it opens in a new tab: https://github.com/argoproj/argo-workflows/blob/main/ui/src/app/shared/components/links.tsx#L38

Ideally there should be an open to make the link do this by default

**Reply by agilgur5** [reply]
Yes it's been discussed before too [on Slack](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1707989479129399) and https://github.com/argoproj/argo-workflows/issues/11112#issuecomment-1746988447, would you like to file a feature request for this?

**Reply by agilgur5** [reply]
I opened a feature request: https://github.com/argoproj/argo-workflows/issues/13114
